core/runtime.rs

```diff
@@ -3456,7 +3532,99 @@ pub mod tests {
     let _ = runtime.mod_evaluate(module_id);
     futures::executor::block_on(runtime.run_event_loop(false)).unwrap();
 
-    let _snapshot = runtime.snapshot();
+    {
+      let module_map_rc = runtime.get_module_map();
+      let module_map = module_map_rc.borrow();
+      assert_eq!(module_map.ids_by_handle.len(), 1);
+      assert_eq!(module_map.ids_by_handle.values().next().unwrap(), &1);
+      assert_eq!(module_map.handles_by_id.len(), 1);
+      assert_eq!(module_map.handles_by_id.keys().next().unwrap(), &1);
+      assert_eq!(module_map.info.len(), 1);
+      assert_eq!(module_map.info.keys().next().unwrap(), &1);
+      assert_eq!(
+        module_map.info.values().next().unwrap(),
+        &ModuleInfo {
+          id: 1,
+          main: true,
+          name: "file:///main.js".to_string(),
+          requests: vec![],
+          module_type: ModuleType::JavaScript,
+        }
+      );
+      assert_eq!(module_map.by_name.len(), 1);
+      assert_eq!(
+        module_map.by_name.keys().next().unwrap(),
+        &(
+          "file:///main.js".to_string(),
+          AssertedModuleType::JavaScriptOrWasm
+        )
+      );
+      assert_eq!(
+        module_map.by_name.values().next().unwrap(),
+        &SymbolicModule::Mod(1)
+      );
+      assert_eq!(module_map.next_module_id, 2);
+      assert_eq!(module_map.next_load_id, 2);
+    }
```

Comment:
Extract to helper function that takes `JsRuntime` as an argument and does all these asserts